src/navigation/PhotosNavigator.tsx

```diff
@@ -18,7 +20,10 @@ function PhotosNavigator(): JSX.Element {
   useEffect(() => {
     photosCtx.permissions.getPermissionsStatus().then((status) => {
       setPermissionsStatus(status);
-      setIsInitialized(true);
+
+      setTimeout(() => {
```

Comment:
Is this for UX purposes or there is something apart going on here?

Reply:
is just an UX tweak, so you don't face a white screen when navigating to the Photos gallery